style/ruby/README.md

```diff
@@ -40,8 +40,8 @@ Ruby
 * Use a trailing comma after each item in a multi-line list, including the last
   item. [Example][trailing comma example]
 * Use heredocs for multi-line strings.
-* Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
-  and `attr_accessor`s.
+* Use `private` for non-public `attr_reader`s, `attr_writer`s, and
```

Comment:
Can we say `Prefer` instead of `Use`?

You have to use `protected` if you want to refer to another instance's attributes in methods like `==` or `hash`.